src/runtime/start/start-module.ts

```diff
@@ -58,41 +63,50 @@ export function createStartModuleContent(config: StartModuleConfig): string {
     })
   `
 
-  if (config.relativePackageJsonPath) {
+  if (config.layout.packageJson) {
     content += EOL + EOL + EOL
     content += stripIndent`
       // package.json is needed for plugin auto-import system.
       // On the Zeit Now platform, builds and dev copy source into
       // new directory. Copying follows paths found in source. Give one here
       // to package.json to make sure Zeit Now brings it along.
-      require('${config.relativePackageJsonPath}')
+      require('${
+        config.absoluteModuleImports
```

Comment:
Don't think we need this condition here?

But, we do need to use it to know if we need to use `requireFrom`.

Keeping this code always present I know isn't needed, but its simpler if we do, and in this case acceptable little addition.

Reply:
Sorry, not sure I follow what you mean

Reply:
nevermind!